client/src/components/EmailAlertSignup.tsx

```diff
@@ -59,12 +62,12 @@ const BuildingSubscribeWithoutI18n = (props: BuildingSubscribeProps) => {
       <Trans render="div" className="status-description">
         Click the link we sent to {email}. It may take a few minutes to arrive.
       </Trans>
-      <Trans render="div">Didn’t get the link?</Trans>
-      <Button
+      {!verifyResent && <Trans render="div">Didn’t get the link?</Trans>}
+      <SendNewLink
+        linkSent={verifyResent}
+        setLinkSent={setVerifyResent}
```

Comment:
```suggestion
        setParentState={setIsEmailResent}
```

Reply:
`linkSent` does't have to be passed into `SendNewLink`,  since the start state is always going to be false